2021q4/doceng.adoc

```diff
@@ -0,0 +1,74 @@
+////
+Quarter:	4th quarter of 2021
+Prepared by:	dbaio
+Reviewed by:	carlavilla, bcr
+Last edit:	$Date: 2022-01-05 20:07:39 -0300 (Wed, 05 Jan 2022) $
+Version:	$Id: doceng-2021-4th-quarter-status-report.adoc 208 2022-01-05 23:07:39Z dbaio $
+////
+
+=== Documentation Engineering Team
+
+Link: link:https://www.freebsd.org/docproj/[FreeBSD Documentation Project] +
+Link: link:https://docs.freebsd.org/en/books/fdp-primer/[FreeBSD Documentation Project Primer for New Contributors] +
+Link: link:https://www.freebsd.org/administration/#t-doceng[Documentation Engineering Team]
+
+Contact: FreeBSD Doceng Team <doceng@FreeBSD.org>
+
+The doceng@ team is a body to handle some of the meta-project issues associated with the FreeBSD Documentation Project; for more information, see link:https://www.freebsd.org/internal/doceng/[FreeBSD Doceng Team Charter].
```

Comment:
s/is a body to handle/the group handling/ (more natural wording). Or maybe "the team handling", since you call it "team" in the link text on that line.

Reply:
This is from the [FreeBSD Doceng Team Charter](https://www.freebsd.org/internal/doceng/). Let's keep it that way.